ome/omero-matlab
# Gradle

To test: 
* Run ``gradle build``
* Check that a zip is generated under ``build/distributions``
* Select the zip, unzip
* Add the folder and subfolder to the Matlab path
* Check that you can load OMERO and connect to the server
You can try a "full workflow" using scripts from https://github.com/ome/training-scripts/tree/master/practical/matlab

Note that currently all the jars included under libs are added to the java classpath.
This could be filtered.
B-F ones are now added, this should allow us to include the approach followed by @kouichi-c-nakamura 

The zip will be available on GitHub when a tag is pushed
